### PR TITLE
Adding assets / Minor fixes

### PR DESCRIPTION
**Pull request by b0llu**

### CODE REVIEW

File: ecom-css/section.css [hunk 1]
```diff
@@ -25,4 +25,13 @@ section .hamburger {
...
+}
+
+::-webkit-scrollbar {
+    width: 5px;
+}
+
+::-webkit-scrollbar-thumb {
+    background: var(--primary-color);
+    border-radius: 10px;
 }
```
Comment (manojsarna) [inline ecom-css/section.css:37]: looks good!!

File: landing.css [hunk 2]
```diff
@@ -158,6 +171,19 @@ footer {
...
+@media screen and (max-width: 1500px) {
+    .hero-img {
+        width: 90%;
+    }
+}
+
+@media screen and (max-width: 1000px) {
+    .hero-img {
+        width: 100%;
+        height: 50rem;
```
Comment (manojsarna) [inline landing.css:174]: looks good!!

**Review by manojsarna — APPROVED** [review]
overall looks good

check with the bug fixes!!

**Review by vashnavichauhan1825 — COMMENTED** [review]
everything is fine !

**Pull request merged**